### PR TITLE
Grammatical corrections for descriptions

### PR DESCRIPTION
**Pull request by lessw2020**
Just updated a bunch of mis-spellings and grammatical mistakes.
Note I ran nbstrip and that seems to be why the 'output' are missing?  
Not sure if there's a way to just take my sentence corrections, but dealing with nb's seems hard so far.

### CODE REVIEW

**Comment by jph00** [comment]
It looks like some of the show_doc output is being removed in this diff. Can you put it back - and in general try to keep your PR diffs as clean as you can, because it really helps us review them. :)

**Comment by lessw2020** [comment]
Hi - point noted,  but the issue is this.  When I open the notebook on my salamander server, the output does not show.  Instead it just shows:
<IPython.core.display.Markdown object>
Instead of the doc text.  
I didn't notice it but that's why it has two blocks missing when I check in.
When I try to execute the show_doc, I get this error:
execute:
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-1-46207f511c13> in <module>
----> 1 show_doc(Optimizer.zero_grad)

NameError: name 'show_doc' is not defined
---------------------------------------------------
So basically it seems I can't do anything in the notebook at this point or these executed show docs are going to be missing as they aren't there when I just load it from master.

This may be a bigger issue that I can't run v2 generically (posted about that) but unclear why show_doc which is Jupyter would not display here.
Anyway go ahead and close this.  I'll see if I can hand edit in JSON to make the grammer fixes now and figure out the show_doc later.

**Comment by jph00** [comment]
`show_doc` missing means that you haven't run the import cell at the top of the nb yet. Let's figure out on the forums why you can't get things running.
